packages/headless/src/features/facets/range-facets/generic/range-facet-reducers.ts

```diff
@@ -28,6 +28,21 @@ export function registerRangeFacet<T extends RangeFacetRequest>(
   state[facetId] = {...request, numberOfValues};
 }
 
+export function updateRangeValues<T extends RangeFacetRequest>(
```

Comment:
Created a very general action, a direct update of the facet values, without further tempering of the values.

Reply:
With great power comes great responsibility. I was debating whether we should publicly expose such an action. I think it should be fine though,